content/hi/partners/_index.html

```diff
@@ -0,0 +1,91 @@
+---
+title: सहभागी
+bigheader: कूबरनेट्स सहभागी
+abstract: कूबरनेट्स इकोसिस्टम का विस्तार.
+class: gridPage
+cid: सहभागी
+---
+
+<section id="users">
+    <main>
+        <h5>कुबरनेट्स सहयोगियों संग एक मज़बूत कोड बेस का निर्माण करता है जो संपूरक मंचों के स्पेक्ट्रम को सहयोग करता है।</h5>
+        <div class="col-container">
+          <div class="col-nav">
+            <center>
+              <h5>
+                <b>कुबरनेट्स प्रमाणित सेवा प्रदाता</b>
+              </h5>
+              <br>उद्यमों द्वारा कुबरनेट्स सफलतापूर्वक अपनाने में सहभागी अनुभवी सेवा प्रदाता।
+              <br><br><br>
+              <button id="kcsp" class="button" onClick="updateSrc(this.id)">KCSP सहभागी देखें</button>
+              <br><br><a href="https://www.cncf.io/certification/kcsp/">KCSP बनने में दिलचस्प</a>?
+            </center>
+          </div>
+          <div class="col-nav">
+            <center>
+              <h5>
+                <b>प्रमाणित कूबरनेट्स Distributions, Hosted Platforms, तथा Installers</b>
+              </h5>सॉफ्टवेयर अनुरूप चलने से ये सुनिश्चित किया जाता है कि प्रत्येक विक्रेता का संस्करण जरूरी API का समर्थन करे।
+              <br><br><br>
+              <button id="conformance" class="button" onClick="updateSrc(this.id)">अनुरूप सहभागी देखें</button>
+              <br><br><a href="https://www.cncf.io/certification/software-conformance/">प्रमाणित कूबरनेट्स बनने में दिलचस्प</a>?
+            </center>
+          </div>
+          <div class="col-nav">
+            <center>
+              <h5><b>कूबरनेट्स प्रशिक्षण सहभागी</b></h5>
```

Comment:
may be: कुबेरनेट्स प्रशिक्षण सहभागियों

Reply:
I don't think so, will wait for others to review.

Reply:
This sentence `कुबेरनेट्स प्रशिक्षण सहभागियों` is not matching up @Rajakavitha1

Reply:
`सहभागी` or `सहभागियों` is not really helping here, how about `कूबरनेट्स प्रशिक्षण प्रदाता` means Kubernetes training provider.

Reply:
It is actually "Kubernetes Training Partners" in the English version, I think  प्रदाता (Provider) will change the whole meaning.

Reply:
I know it will change the sentence but the meaning should be the same, **these are the trainers who provide Kubernetes training**.